lib/regaliator/configuration.rb

```diff
@@ -20,7 +19,7 @@ def initialize
     end
 
     def secure?
-      use_ssl == true
```

Comment:
I'd be fine keeping it as is to only explicitly allow `true` instead of anything "truthy", unless there is a good reason to be more lenient?